cvmfs/util/algorithm.h

```diff
@@ -108,6 +115,59 @@ class StopWatch : SingleCopy {
 };
 
 
+class Log2Hist
+{
+ private:
+  uint number_of_bins;
+  atomic_int32 *bins;
+  uint *boundary_values;
+  uint count_digits(uint64_t n)
+  {
+    return (uint)floor(log10(n) + 1);
+  }
+  string generate_stars(uint n)
```

Comment:
Actually, this can be achieved simply by the std::string constructor that fills the string with a sequence of (equal) chars

Reply:
Great idea! It did not came to my mind.